paperclip.gemspec

```diff
@@ -24,8 +24,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency("activemodel", ">= 4.2.0")
   s.add_dependency("activesupport", ">= 4.2.0")
-  s.add_dependency("mime-types")
```

Comment:
I think that the `mime-types` dependency is still needed due to [this require in `lib/paperclip.rb`](https://github.com/kreeti/kt-paperclip/pull/54/files#diff-31e78c641c3ccf9fd8438aeb0d530fb7b9dc400878d3e77973948d4f175d4c94R64). At least I wasn't able to start rails with this patch without the gem installed.

Reply:
Hmmmm, you might be right. It must have worked for me because I have other deps that require `mime-types`.

I don't understand how the tests are passing without this. Any ideas?

Reply:
Thanks, it looks that the `mime-types` gem is indirectly required through cucumber. The Travis log shows that the `mime-types` gem is [indeed installed](https://travis-ci.com/github/kreeti/kt-paperclip/jobs/494334560#L446) and cucumber is [one of its dependants](https://rubygems.org/gems/mime-types/reverse_dependencies?rdeps_query=cucumber).

Reply:
Nice pickup. Makes sense why it works in CI but would break in the field. Can you please check if https://github.com/kreeti/kt-paperclip/pull/54/commits/4347fab325033078abe26e68af03fd65d971054e fixes things for you?

Reply:
It does! Thank you Alex.